MonoGame.Framework/Graphics/GraphicsAdapter.cs

```diff
@@ -107,6 +111,12 @@ public DisplayMode CurrentDisplayMode
                     graphics.ReleaseHdc(dc);
                     return new DisplayMode(width, height, SurfaceFormat.Color);
                 }
+#elif WINDOWS_UAP
+                var visibleBounds = ApplicationView.GetForCurrentView().VisibleBounds;
+                var rawPixelsPerViewPixel = DisplayInformation.GetForCurrentView().RawPixelsPerViewPixel;
+                return new DisplayMode((int)(visibleBounds.Width * rawPixelsPerViewPixel + 0.5f),
+                                       (int)(visibleBounds.Height * rawPixelsPerViewPixel + 0.5f),
+                                       SurfaceFormat.Color);
 #else

```

Comment:
100% agreed with @tomspilman .  :) Just note you're getting the resolution of monitor 0 on card 0...one could have multiple resolutions across cards/monitors...